discovery-provider/src/queries/trending.py

```diff
@@ -12,160 +12,34 @@
 from src.utils.config import shared_config
 from src.queries import response_name_constants
 from src.queries.query_helpers import get_repost_counts, get_pagination_vars, get_save_counts, get_genre_list
+from src.tasks.generate_trending import generate_trending
 
 logger = logging.getLogger(__name__)
 bp = Blueprint("trending", __name__)
 
+REDIS_URL = shared_config["redis"]["url"]
```

Comment:
@hareeshnagaraj not blocking on this PR but we init lib's like in two or three different files in the server. we should have a singleton for redis access

Reply:
agreed